topics/assembly/tutorials/post-assembly-quality-control/tutorial.md

```diff
@@ -0,0 +1,411 @@
+---
+layout: tutorial_hands_on
+
+title: Post Assembly Quality Control Workflow
+zenodo_link: ''
+questions:
+- what combination of tools can assess the quality of an initial assembly?
+- how to use the different tools in galaxy?
+objectives:
+- apply the post-assembly-QC-workflow using the necessary tools
+- understand the function of the different tools
+time_estimation: 3H
+key_points:
+- The take-home messages
+- They will appear at the end of the tutorial
+contributors:
+- contributor1
+- contributor2
+
+---
+
+
+# Introduction
+
+An important part in genome assembly is quality control. During the whole process of DNA sequencing and assembling the genome, errors like mismatches and gaps can occur. This can affect the accuracy and completeness of the genome. Quality control helps to identify and correct these errors by evaluating the quality of the sequences and by detecting and removing potential contaminations. {% cite DominguezDelAngel2018 %}
+A high-quality end-product as a result also ensures to avoid errors and false conclusions in downstream analyses.
+
+Since there are many different ways how errors can occur there are also many different tools to identify and remove potential problems. The difficult part is to choose between them and to know when it's time to move on. It is important because time and resources play a big role in genome assembly.
+
+But how can we gauge the quality of an assembly? To answer this question we first have to find out the properties to evaluate the quality of an assembly. It turned out to be a good idea to focus on the following aspects: contiguity, completeness and correctness. To gauge those properties some metrics are needed. It is necessary to aim for a value like a score to have the information of the measured quality as one number. This is useful in order to have an easy overview and make it easier for comparisons.({% cite Wang2023 %})
+
+Length metrics like N50, CC ratio and sizes of contigs, scaffolds and gaps are common for contiguity analysis. Tools like gfastats will provide these kinds of metrics and other statistical values by a given assembly to assess contiguity quality. 
+BUSCO and genomescope will help us to evaluate the completeness of the genome. BUSCO for example compares the assembly to a set of conserved single-copy genes and the other tool generates a k-mer profile analysis giving us information about the coverage to frequency ratio.
+Correctness or accuracy of an assembly can be gauged by checking base-level and structural errors. This can be done by mapping the raw sequencing reads to the target assembly and then calculating the error rate or by comparing the assembly to a set of k-mers from raw sequencing reads. Therefore we will use blobtoolkit and merqury.
```

Comment:
I would introduce each tool in their specific section, instead of including that information in the introducion.